File: CD Ultima Entrega/Fuentes/Frontend/src/app/shared/servicio.ts
import { Utilidades } from "./utilidades";
import { Pregunta } from "./pregunta";

export class Servicio {
    Id: number;
    Nombre: string;
    Imagen: string;
    Habilitado: boolean;
    FechaCreacion: Date;
    Preguntas : Pregunta[] = [];
    

    constructor() {      
        // this.Barrio = new Barrio();
        // this.Barrio.Id=0;
    }
}

File: CD Entrega Vieja/Fuentes/Backend y API Rest/ET/Servicio.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ET
{
    public class Servicio
    {
        public int Id { get; set; }
        public string Nombre { get; set; }
        public string Imagen { get; set; }
        public bool Habilitado { get; set; }
        public DateTime FechaCreacion { get; set; }
        public List<Pregunta> Preguntas { get; set; }

        public Servicio() {
            Preguntas = new List<Pregunta>();
        }
    }
}
